src/arduino-rfm/LoRaMAC.cpp

```diff
@@ -39,6 +39,10 @@
 #include "Config.h"
 #include "Arduino.h"
 
+#define AU_915 // use the band youy need

```

Comment:
The library specifies that this needs to be defined on the file Config.h not in the LoRaMAC.cpp